packages/switch/src/LionInputSwitch.js

```diff
@@ -0,0 +1,63 @@
+import { html } from '@lion/core';
+import { LionField } from '@lion/field';
+import { ChoiceInputMixin } from '@lion/choice-input';
+
+import '../lion-button-switch.js';
+
+export class LionInputSwitch extends ChoiceInputMixin(LionField) {
+  get slots() {
+    return {
+      ...super.slots,
+      input: () => document.createElement('lion-button-switch'),
+    };
+  }
+
+  render() {
+    return html`
+      <div class="switch__slots">
+        <slot name="label"></slot>
+        <slot name="help-text"></slot>
+        <slot name="feedback"></slot>
+      </div>
+      <slot name="input"></slot>
+    `;
+  }
+
+  connectedCallback() {
+    super.connectedCallback();
+    this._setupButtonSwitch();
+    this._syncButtonSwitch();
+  }
+
+  disconnectedCallback() {
+    super.disconnectedCallback();
+    this._cleanButtonSwitch();
+  }
+
+  updated(changedProperties) {
+    super.updated(changedProperties);
+    this._syncButtonSwitch();
+  }
+
+  get _buttonSwitch() {
```

Comment:
As a convention: getters like these are usually suffixed with `Node`. 
Kind of means they refer functionally important nodes in your component and Subclassers are allowed to either call or override those getters.

Something we should definitely document somewhere ;)

Reply:
updated, please confirm and "Resolve conversation"